triple-store/triple-store-api/src/main/java/com/powsybl/triplestore/api/PropertyBag.java

```diff
@@ -19,14 +21,15 @@
  */
 public class PropertyBag extends HashMap<String, String> {
 
-    public PropertyBag(List<String> propertyNames) {
-        this(propertyNames, true);
+    public PropertyBag(List<String> propertyNames, boolean unescapeIdentifiers) {
```

Comment:
Maybe use a factory method instead of adding a boolean to the constructor, to state the behaviour of the property bag ? `PropertyBag.createWithEscapedIdentifiers()`

Reply:
Or `PropertyBag.createForEscapedIdentifiers()` ?

Reply:
Same question, is `escapeIdentifiers` okay for you?

Reply:
Same thing, `decodeEscapedIdentifiers`?